src/components/Card/styled.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { Card as BoostrapCard, Col, Container } from 'react-bootstrap';
+
+const Img = styled.img`
```

Comment:
I suggest you export each styled component individually, so when you use it, instead of calling `<Styled.Img />` just use `<Img />`. Code looks cleaner that way.